src/data/events.json

```diff
@@ -0,0 +1,21 @@
+{
+	"events": [
+		{
+			"type": "social",
+			"title": "Casual Catchup",
+			"date": "2018-08-08T10:00:00.000Z",
```

Comment:
Store the dates as dates and get GraphQL to convert it to the format needed. Likely what will return from Meetup is something similar to this

Reply:
https://secure.meetup.com/meetup_api/console/?path=/:urlname/events

urlname = Front-End-Web-Developers-Perth